src/main/kotlin/com/example/bitcoin_multisig_coordinator_server/BitcoinMultiSigCoordinatorServerApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.bitcoin_multisig_coordinator_server
```

Comment:
Let's use `bitcoindevkit.org` for the group and `karavan` for the app name. You'll need to also change the directory to `org/bitcoindevkit/karavan` and this file name to `Karavan.kt`:
```suggestion
package org.bitcoindevkit.karavan
```

Reply:
Double check if I did this correct. I changed directory to org.bitcoindevkit/karavan instead of org/bitcoindevkit/karavan because the latter was invalid.